### PR TITLE
St relate

### PR DESCRIPTION
**Pull request by Robinlovelace**

### CODE REVIEW

**Comment by Robinlovelace** [comment]
Plan for this: merge and then create a separate PR adding a new section on DE-9IM. Looking at #699 there's clearly enough content and justification for a new section.

**Comment by Robinlovelace** [comment]
Result from previous commit

image

File: 04-spatial-operations.Rmd [hunk 1]
```diff
@@ -316,15 +305,14 @@ plot(l, add = TRUE)
...
-<!-- Todo:reinstate this section (RL 2021-11) -->
-<!-- ### DE-9IM strings -->
+### DE-9IM strings
 
-<!-- Underlying the binary predicates demonstrated in the previous section is the Dimensionally Extended 9-Intersection Model (DE-9IM). -->
-<!-- The model was originally labelled "DE + 9IM" by its inventors, referring to the "dimension of the intersections of' boundaries, interiors, and exteriors of two features" [@clementini_comparison_1995] -->
-<!-- , but is now generally referred to as DE-9IM [@shen_2018_classification] -->
-<!-- The way the model works can be demonstrated with reference to two intersecting polygons, as illustrated in Figure \@ref(fig:de-9im). -->
+Underlying the binary predicates demonstrated in the previous section is the Dimensionally Extended 9-Intersection Model (DE-9IM).
+The model was originally labelled "DE + 9IM" by its inventors, referring to the "dimension of the intersections of' boundaries, interiors, and exteriors of two features" [@clementini_comparison_1995], but is now referred to as DE-9IM [@shen_classification_2018].
```
Comment (Nowosad) [inline 04-spatial-operations.Rmd:311]: of' ?

**Comment by Robinlovelace** [comment]
Thanks @Nowosad. Good point, work in progress, will ping when ready.

**Comment by Robinlovelace** [comment]
Heads-up @Nowosad this is ready to go I think. We can tidy up the details after but it's looking good from the output of `bookdown::serve_book()`: 

image

**Comment by Robinlovelace** [comment]
Going to merge. Can you take a look at the rendered version and make further tweaks?

**Comment by Nowosad** [comment]
Hi @Robinlovelace -- feel free to merge it. We can edit it later

**Pull request merged**